### PR TITLE
Fix references to docker-compose in comments

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description ##

This pull request updates this repository to use the `docker compose` syntax instead of the `docker-compose` syntax.

## 💭 Motivation and context ##

The `docker compose` syntax is the preferred (and only correct) syntax after the changes in cisagov/ansible-role-docker#60.

## 🧪 Testing ##

Automated testing passes.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated to reflect the changes in this PR.
- [x] All new and existing tests pass.

### CODE REVIEW

File: src/assessor_portal.yml [hunk 1]
```diff
@@ -17,9 +17,9 @@
             chdir: "{{ assessor_portal_dir }}"
             cmd: "{{ assessor_portal_dir }}/run.py pull"
         - name: Pull images from Assessor Portal docker-compose.yml
-          community.docker.docker_compose:
-            project_src: "{{ assessor_portal_dir }}"
-            pull: yes
+          ansible.builtin.command:
+            chdir: "{{ assessor_portal_dir }}"
+            cmd: docker compose pull
```
Comment (dav3r) [inline src/assessor_portal.yml:26]: Is it worth adding a comment in here that as of now, there is no Ansible module that supports `docker compose` commands, though there likely will be one at some point in the future?

**Review by dav3r — APPROVED** [review]
👍

File: src/assessor_portal.yml (hunk 1 above)
Comment (jsf9k) [reply src/assessor_portal.yml:26]: Please see commit 2e581b1.

Comment (jsf9k) [reply src/assessor_portal.yml:26]: @dav3r - Can this conversation be resolved?

Comment (dav3r) [reply src/assessor_portal.yml:26]: Yes, sorry for not circling back to this.

File: src/assessor_portal.yml [hunk 2]
```diff
@@ -16,10 +16,14 @@
           ansible.builtin.command:
             chdir: "{{ assessor_portal_dir }}"
             cmd: "{{ assessor_portal_dir }}/run.py pull"
+        # The community.docker.docker_compose module exists, but it
+        # only supports docker-compose and not the Docker compose
```
Comment (mcdonnnj) [inline src/assessor_portal.yml]: Just lining up with their documentation.

```suggestion
        # only supports docker-compose and not the Docker Compose
```

**Review by mcdonnnj — APPROVED** [review]
Makes sense. I have one minor capitalization change but otherwise LGTM ✔

**Pull request merged**